openverse_catalog/dags/providers/provider_api_scripts/jamendo.py

```diff
@@ -184,27 +184,50 @@ def _get_foreign_identifier(media_data):
         return None
 
 
-def _get_audio_info(media_data):
+def _remove_param_from_url(url: str, param: str) -> str:
+    """
+    Remove a parameter from a provided URL.
+    """
+    parsed_url = urlsplit(url)
+    query = parse_qs(parsed_url.query)
+    query.pop(param, None)
+    return parsed_url._replace(query=urlencode(query, doseq=True)).geturl()
+
+
+def _remove_from(audio_url: Optional[str]) -> Optional[str]:
+    """
+    Remove the "from" parameter from an audio URL. Audio URLs have a "from" param which
+    seems to encapsulate information about the calling application.
+    Example from the API:
+    https://prod-1.storage.jamendo.com/?trackid=1532771&format=mp31&from=app-devsite
+    This information looks like an API key or secret when returned, so we remove it
+    since it's not necessary for serving the audio files.
+    >>> base_url = "https://prod-1.storage.jamendo.com/"
+    >>> url = f"{base_url}?trackid=1532771&format=mp31&from=app-devsite"
+    >>> _remove_from(url)
+    'https://prod-1.storage.jamendo.com/?trackid=1532771&format=mp31'
+    """
+    if audio_url is None:
+        return
+    return _remove_param_from_url(audio_url, "from")
+
+
+def _get_audio_info(media_data: dict) -> tuple[Optional[str], int]:
     """Parses audio URL, audio download URL, audio duration
     If the audio does not allow download, we save the 'streaming'
     URL as the `audio_url`
```

Comment:
This part of the docstring should be updated.

Optionally, it seems reasonable to me to have the documentation for removing the 'from' parameter here and remove the `_remove_from` function, since `_get_audio_info` is itself very simple now. Total bikeshedding, but `_remove_from` looks confusing to me, I think because 'from' is a preposition and the documentation is in a different function.

`_remove_trackid` makes more sense to me and I'd be fine keeping it even if you do decide to remove `_remove_from`. Up to you though!